ibm/mas_devops/roles/aibroker_tenant/templates/aibroker/aibrokerworkspace.yml.j2

```diff
@@ -21,6 +21,12 @@ spec:
             url: "{{ drocfg.url }}"
             secretName: "{{ mas_aibroker_dro_token_secret }}"
             ca: "{{ drocfg.ca }}"
+
+        rsl:
+            url: "{{ rslcfg.url }}"
+            token : "{{ rslcfg.rsl_token }}"
```

Comment:
The token should not be included here as it is a secret value. There should be a field called `secretName` whose value is `{{ mas_aibroker_rsl_token_secret }}`.

Reply:
The field names also need to match those in https://github.ibm.com/maximoappsuite/ibm-mas-aibroker/pull/156